lms/services/lti_names_roles.py

```diff
@@ -0,0 +1,56 @@
+"""
+Service to talk to the Name and Roles LTIA API
+
+https://www.imsglobal.org/spec/lti-nrps/v2p0
+"""
+from typing import List, TypedDict, Optional
+from lms.services.ltia_http import LTIAHTTPService
+
+
+Member = TypedDict(
```

Comment:
I reckon this is a good documentation tool and we get more value out of it than just pasting an example record in the method's docstring.

Reply:
So we don't actually use this?

I kind of prefer the look of `dataclass`s, especially if it's just for show?